controller/src/test/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessorsTest.java

```diff
@@ -106,21 +105,30 @@ public void testHandleOrphaned() throws CheckpointStoreException {
         EventProcessorSystem system = mock(EventProcessorSystem.class);
         EventProcessorGroup<ControllerEvent> processor = getProcessor();
         EventProcessorGroup<ControllerEvent> mockProcessor = spy(processor);
+        CuratorZookeeperClient curatorZKClientMock = mock(CuratorZookeeperClient.class);
 
         doThrow(new CheckpointStoreException("host not found")).when(mockProcessor).notifyProcessFailure("host3");
 
         when(system.createEventProcessorGroup(any(), any(), any())).thenReturn(mockProcessor);
-
+        when(checkpointStore.isHealthy()).thenReturn(false).thenReturn(true);
         @Cleanup
         ControllerEventProcessors processors = new ControllerEventProcessors("host1",
                 config, localController, checkpointStore, streamStore, bucketStore, 
                 connectionPool, streamMetadataTasks, streamTransactionMetadataTasks, 
                 kvtStore, kvtTasks, system, executorService());
-        //check for a case where init is not initalized so that kvtRequestProcessors don't get initialized and will be null
-        assertTrue(Futures.await(processors.sweepFailedProcesses(() -> Sets.newHashSet("host1"))));
-        Assert.assertFalse(processors.isReady());
-        Assert.assertFalse(processors.isBootstrapCompleted());
-        Assert.assertFalse(processors.isMetadataServiceConnected());
+        ControllerEventProcessors spyProcessors = spy(processors);
```

Comment:
Please reverse these changes and add a new test case, as discussed.

Reply:
done